themes/tcars/theme.css

```diff
@@ -0,0 +1,420 @@
+@import url('../../fonts/pixelsagas.com/PS_Okuda/font.css');
+@import url('../../fonts/LCARS/font.css');
+
+:root,
+::backdrop {
+    --main-color-bg: hsl(200deg 30% 3%); /* deep temporal void */
+    --main-color-text: hsl(180deg 70% 85%); /* bright chroniton cyan */
+    --nav-color-bg: var(--main-color-text);
+    --nav-color-text: var(--main-color-bg);
+    --main-font-body: "Okuda", sans-serif;
+    --main-font-headers: "Okuda", sans-serif;
+    --shadow-color-text: hsl(180deg 60% 25%);
+    --dialog-color-backdrop: hsl(200deg 50% 5% / 80%);
+    --main-a-link-color: hsl(180deg 65% 55%);
+    --main-a-hover-color: hsl(180deg 75% 65%);
+    --main-a-active-color: hsl(180deg 55% 45%);
+    --main-a-visited-color: hsl(190deg 50% 60%);
+    --main-font-size: 14pt;
+    --void-bg: hsl(200deg 40% 2%);
+    --min-decoration-height: 28em;
+    --min-body-height: calc(1em + var(--min-decoration-height));
+    
+    /* Temporal color palette - chroniton blues and quantum golds */
+    --trek-color-1: hsl(180deg 80% 60%); /* bright chroniton cyan */
+    --trek-color-2: hsl(190deg 70% 55%); /* temporal blue */
+    --trek-color-3: hsl(45deg 85% 60%); /* quantum gold */
+    --trek-color-4: hsl(280deg 60% 65%); /* temporal violet */
+    --trek-color-5: hsl(170deg 75% 50%); /* phase teal */
+    --trek-color-6: hsl(200deg 80% 70%); /* holographic light blue */
+    --trek-color-highlight: var(--trek-color-1);
+    
+    /* Player colors with temporal theme */
+    --player-color-red: hsl(10deg 70% 50%);
+    --player-color-blue: hsl(190deg 70% 55%);
+    --player-color-yellow: var(--trek-color-3);
+    --player-color-brown: hsl(30deg 50% 45%);
+    --player-color-black: hsl(200deg 20% 15%);
+    --player-color-white: hsl(180deg 30% 80%);
+    
+    /* Temporal effect variables */
+    --temporal-glow: 0 0 15px var(--trek-color-1), 0 0 25px var(--trek-color-2);
+    --chroniton-shadow: 0 0 20px var(--trek-color-1);
+    --quantum-glow: 0 0 10px var(--trek-color-3);
+}
+
+html.alt-font {
+    --main-font-body: "LCARS", sans-serif;
+    --main-font-headers: "LCARS", sans-serif;
+    --headers-weight: normal;
+    --headers-text-transform: uppercase;
+}
+
+.alt-font-available {
+    display: unset;
+}
+
+body, h1, h2, h3, h4, h5, h6 {
+    letter-spacing: 0.06em;
+    text-shadow: var(--temporal-glow);
+}
+
+body nav {
+    align-items: center;
+}
+
+body {
```

Comment:
let's remove the `body > header` background (set it to transparent) so it doesn't interfere with the glow on the decoration elements.